src/selectors/nodes.test.js

```diff
@@ -370,4 +376,17 @@ describe('Selectors', () => {
       expect(Object.keys(nodesWithInputParams)).toEqual(parameterNodesID);
     });
   });
+
+  describe('getInputOutputDataNodes', () => {
+    it('includes input output ndoes related to a modular pipeline in the returned object', () => {
```

Comment:
typo - nodes

Reply:
You can use github suggestion to suggest changes for these small issues like this:

```suggestion
    it('includes input output nodes related to a modular pipeline in the returned object', () => {
```

Reply:
Thanks both - great spot! I've updated the typo in my latest commit